src/sentry/static/sentry/app/views/issueList/actions.jsx

```diff
@@ -361,18 +405,36 @@ const IssueListActions = createReactClass({
             <Checkbox onChange={this.handleSelectAll} checked={pageSelected} />
           </ActionsCheckbox>
           <ActionSet>
-            <ResolveActions
-              hasRelease={hasReleases}
-              latestRelease={latestRelease}
-              orgId={orgId}
-              projectId={projectId}
-              onUpdate={this.handleUpdate}
-              shouldConfirm={this.shouldConfirm('resolve')}
-              confirmMessage={confirm('resolve', true)}
-              confirmLabel={label('resolve')}
-              disabled={resolveDisabled}
-              disableDropdown={resolveDropdownDisabled}
-            />
+            {selectedProjectSlug ? (
+              <Projects orgId={orgId} slugs={[selectedProjectSlug]}>
+                {({projects, initiallyLoaded, fetchError}) => {
+                  const selectedProject = projects[0];
+                  return initiallyLoaded ? (
+                    fetchError ? (
+                      <LoadingError message={t('Error loading project')} />
+                    ) : (
+                      this.renderResolveActions({
+                        hasReleases: new Set(selectedProject.features).has('releases'),
+                        latestRelease: selectedProject.latestRelease,
+                        projectId: selectedProject.slug,
+                        confirm,
+                        label,
+                      })
+                    )
+                  ) : (
+                    <LoadingIndicator mini />
```

Comment:
not too sure how i feel about having a loading indicator here, or what the appropriate loading should feel like. Perhaps we could move the loading prop deeper to just have the dropdown be loading?

Reply:
Yeah I don't like having the indicator there, seems pretty disruptive, maybe just disable the button while it's loading instead?

Reply:
ill keep the resolve button enabled, but just disable the dropdown since that is what the project info is used for.